todoman/ui.py

```diff
@@ -24,19 +24,23 @@ class TodoEditor:
     The UI for a single todo entry.
     """
 
-    def __init__(self, todo, lists, formatter):
+    def __init__(self, todo, lists, formatter,current_list):
```

Comment:
No `current list` is being passed when creating new `TodoEditor` instances (so this crashes).

I think you didn't mean to keep this bit.